src/auto_contiguous.cpp

```diff
@@ -59,17 +60,24 @@ void auto_contiguous::apply(module& m) const
     auto last = std::prev(m.end());
     for(auto ins : iterator_for(m))
     {
-        if(ins->name() == "layout")
+        if(contains({"layout", "contiguous", "@return", "@param", "@outline"}, ins->name()))
```

Comment:
We shouldn't be skipping `@param`, is there a test that fails for that?

I dont think `@outline` is used anywhere.